### PR TITLE
Add badges and usage info into readme

### LINKED ISSUE #15: Add badges and usage info into README
## Background
README.md is missing some important information

## Feature
Add badges and information how to include the library in projects

### PR DESCRIPTION
**Pull request by benedeki**
Adding some information about the latest release, and usage of the library.

Closes #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +45,46 @@ within the application.**
...
+</dependency>
+<dependency>
+<groupId>za.co.absa.fa-db</groupId>
+<artifactId>slick_2.11</artifactId>
+<version>${latest_version}</version>
+</dependency>
+```
+
+### Scala 2.12 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa/fa-db_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa/fa-db_2.12)
+
```
Comment (dk1844) [inline README.md:78]:
```suggestion
### Scala 2.12 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.fa-db/core_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa.fa-db/core_2.12)

```

Again, just fixed the links here.

File: README.md [hunk 2]
```diff
@@ -30,3 +45,46 @@ within the application.**
...
+libraryDependencies ++= Seq(
+  "za.co.absa.fa-db" %% "core"  % "X.Y.Z",
+  "za.co.absa.fa-db" %% "slick" % "X.Y.Z"
+)
+```
+
+#### Maven
+
+##### Scala 2.11 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa/fa-db_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa/fa-db_2.11)
+
```
Comment (dk1844) [inline README.md:63]:
```suggestion
### Scala 2.11 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.fa-db/core_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa.fa-db/core_2.11)

```

If we want to be technically correct, then you should probably include badges/links both to core & slick artifacts. But for the general sense, this is probably fine (expected that both artifacts would be available always together).

I just fixed the links.

**Review by dk1844 — APPROVED** [review]
LGTM when the links are fixed.

**Review by AdrianOlosutean — APPROVED** [review]
Tried locally

**Pull request merged**